src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -40,13 +41,12 @@
       <_NETStandardLibraryPackageVersion>$(NETStandardLibraryRefPackageVersion)</_NETStandardLibraryPackageVersion>
       <_NETCorePlatformsPackageVersion>$(MicrosoftNETCorePlatformsPackageVersion)</_NETCorePlatformsPackageVersion>
 
-      <!-- TODO: Once .NET 6.0 has released, update these version numbers to 6.0.0 -->
-      <_NET60RuntimePackVersion>6.0.0-rc.2.21452.2</_NET60RuntimePackVersion>
-      <_NET60TargetingPackVersion>6.0.0-rc.2.21452.2</_NET60TargetingPackVersion>
-      <_WindowsDesktop60RuntimePackVersion>6.0.0-rc.2.21431.3</_WindowsDesktop60RuntimePackVersion>
-      <_WindowsDesktop60TargetingPackVersion>6.0.0-rc.2.21431.3</_WindowsDesktop60TargetingPackVersion>
-      <_AspNet60RuntimePackVersion>6.0.0-rc.2.21452.1</_AspNet60RuntimePackVersion>
-      <_AspNet60TargetingPackVersion>6.0.0-rc.2.21452.1</_AspNet60TargetingPackVersion>
+      <_NET60RuntimePackVersion>6.0.$(VersionFeature60)</_NET60RuntimePackVersion>
```

Comment:
I think all of these should be moved to Version.props since they're all expected to service more frequently.
See
https://github.com/dotnet/installer/blob/943ee4f9684b961da16b51888fa09aa66b69f6b5/eng/Versions.props#L75

Reply:
These don't modify with darc so I'd probably not put them in versions.props. Perhaps there should be a ManualVersions.props in the future for all manually updated versions but not for this PR.